### PR TITLE
Apply Decred Color to terminal ui

### PR DESCRIPTION
**Pull request by raphaelint**
Applying Decred color and improving the design of the terminal ui

### CODE REVIEW

File: terminal/pages/history.go [hunk 1]
```diff
@@ -1,16 +1,18 @@
 package pages
 
 import (
+	"github.com/gdamore/tcell"
 	"github.com/rivo/tview"
 )
 
 func HistoryPage() tview.Primitive {
+	color := tcell.NewRGBColor(255, 255, 255)
```
Comment (ademuanthony) [inline terminal/pages/history.go]: seeing that you also used this color in other function below, I think it should be moved up as package level variable. `labelColor` should be a descriptive name

File: terminal/pages/send.go [hunk 2]
```diff
@@ -18,8 +19,8 @@ func SendPage(setFocus func(p tview.Primitive) *tview.Application, clearFocus fu
 	body.AddButton("Cancel", func() {
 		clearFocus()
 	})
-
+	body.SetLabelColor(tcell.NewRGBColor(255, 255, 255))
```
Comment (ademuanthony) [inline terminal/pages/send.go]: use the color created above

File: terminal/terminal.go [hunk 3]
```diff
@@ -49,18 +52,25 @@ func terminalLayout(tviewApp *tview.Application) tview.Primitive {
...
+			changePageColumn(pages.ReceivePage())
 		}).
-		AddItem("Exit", "", 'q', func() {
+		AddItem("Account", "", 'a', nil).
+		AddItem("Security", "", 'x', nil).
+		AddItem("Quit", "", 'q', func() {
 			tviewApp.Stop()
 		})
 	menuColumn.SetCurrentItem(0)
+	menuColumn.SetShortcutColor(tcell.NewRGBColor(112, 203, 255))
```
Comment (ademuanthony) [inline terminal/terminal.go:69]: This color should be created at as package level variable as well. `borderColor` should be a good name

**Review by ademuanthony — CHANGES_REQUESTED** [review]
Good job
Just few observations below

**Comment by itswisdomagain** [comment]
@Sirmorrison please take this up to completion. Discuss with @raedah and @ademuanthony for details of this task.

**Comment by Sirmorrison** [comment]
on it

**Comment by itswisdomagain** [comment]
This solution is completed in #211

**Pull request closed without merging**